src/multi.rs

```diff
@@ -459,6 +426,19 @@ mod tests {
         pb.finish();
     }
 
+    macro_rules! check_order {
```

Comment:
Could this go in a separate PR, or does it depend on the generational token list?

Reply:
It probably could. I'll make it so.